metaversefile-api.js

```diff
@@ -841,6 +841,13 @@ export default () => {
   getAvatarHeight(obj) {
     return getHeight(obj);
   },
+  getGfxSettingJSON(setting){
+    if (setting) return JSON.parse(localStorage.getItem( 'GfxSettings' ))[setting];

```

Comment:
Won't this crash on first load?

Reply:
Good catch.  That line will cause a crash if a specific gfxsettings is requested before the local storage is set. But the functions not necessarily called on first load with a specific setting passed as an arg.  Regardless, needs to be corrected

Reply:
this has now been corrected